integration/log.go

```diff
@@ -24,6 +24,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cenkalti/backoff"
```

Comment:
We've got a client/backoff package. Not sure the server should depend on client code though.